app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -47,6 +47,9 @@ const filterByCapabilities = (filterArray, modelToFilter, isMulti) =>
     return options;
   });
 
+const validateServiceHasResources = (serviceId) => API.get(`/api/storage_services/${serviceId}?attributes=name,storage_resources`)
+  .then((response) => response["storage_resources"].length === 0 ? sprintf(__('Storage service "%s" has no attached storage resources'), response["name"]) : undefined);
```

Comment:
can you try this to avoid lint error - Arrow function used ambiguously with a conditional expression

  ```
.then((response) => (response.storage_resources.length === 0
    ? sprintf(__('Storage service "%s" has no attached storage resources'), response.name)
    : undefined));
```

Reply:
I think additional spaces were used. let's not waste time. I will fix them in a different PR.